mappings/net/minecraft/client/render/model/json/ModelVariantMap.mapping

```diff
@@ -14,6 +14,7 @@ CLASS net/minecraft/class_790 net/minecraft/client/render/model/json/ModelVarian
 	METHOD method_3424 deserialize (Lnet/minecraft/class_790$class_791;Ljava/io/Reader;)Lnet/minecraft/class_790;
 		ARG 0 context
 		ARG 1 reader
+	METHOD method_35791 getVariants ()Ljava/util/Set;
```

Comment:
No, this returns all possible models within this variant map.
```suggestion
	METHOD method_35791 getAllModels ()Ljava/util/Set;
```
Discovered this when trying to map the inner class here.